File: suitframework/templates/tryit/code/call.py
from rulebox.templating import var
from suitframework.lib import helperstry

def currenttime():
    import time
    return time.strftime('%m/%d/%Y %H:%M:%S', time.localtime())

helperstry.currenttime = currenttime
var.username = 'Brandon'
var.loggedin = True
var.members = [
    {
        'name': 'Brandon',
        'admin': True,
        'banned': False
    },
    {
        'name': 'Chris',
        'admin': False,
        'banned': True
    }
]
var.owner = 'Brandon'
var.ownerfull = 'Brandon Evans'

File: suitframework/templates/tryit/code/call.php
<?php
class Helpers
{
    public function currenttime()
    {
        return date('m/d/Y H:M:S');
    }
}

$templating->var->username = 'Brandon';
$templating->var->loggedin = true;
$templating->var->members = array
(
    array
    (
        'name' => 'Brandon',
        'admin' => true,
        'banned' => false
    ),
    array
    (
        'name' => 'Chris',
        'admin' => false,
        'banned' => true
    )
);
$templating->var->owner = 'Brandon';
$templating->var->ownerfull = 'Brandon Evans';
?>